test/object-store/realm.cpp

```diff
@@ -1282,7 +1288,12 @@ TEST_CASE("SharedRealm: convert") {
     SECTION("can copy a synced realm to a synced realm") {
         auto sync_realm1 = Realm::get_shared_realm(sync_config1);
         sync_realm1->begin_transaction();
+        // 'embedded_link' property is null.
```

Comment:
I would have added a separate test for covering this with a separate schema, but that's ok.

Reply:
done